src/sentry/utils/circuit_breaker2.py

```diff
@@ -240,6 +244,11 @@ def record_error(self) -> None:
                     "error_limit_window": controlling_quota.window_seconds,
                 },
             )
+            metrics.incr(
+                f"circuit_breaker.{self.key}.error_limit_hit",
+                sample_rate=1.0,
+                tags={"current_state": state},
```

Comment:
same comment as on the previous PR - I believe we should be able to use `state.value` to get the string value here

Reply:
Yup, good call. Done.